frame/staking-rewards/README.md

```diff
@@ -140,15 +255,52 @@ rolling = min(new_lock - previous_lock, passed_time)
 print(rolling) # time it moves to new lock
 ```
 
+### Expiration
+
+If there is a time-locked position.
+
+After the position expired, a user can unstake without penalty.
+
+If a stake expired, any update on a position will remove its part of the share from the total.
+
+There is a special function that allows anybody to detect expired positions and get a reward for that. Part of ED of the position is transferred to a reporter.
+
 ### Compounding
 
-If fNFT has staked asset as reward asset, it is subject to compounding.
+If a position has staked asset to be the same as a rewarding asset, it is subject to compounding.
```

Comment:
I think we can remove this stuff as it only serves to confuse the readers to mention something we are not implementing

Reply:
We I going to implement it, so it is not confusing. It is matter of less then week to add permission less extrinsic and it will be resonably done. Not automatic so. I think it is good to claim feature and then do (being bullish).